src/playground-code-editor.ts

```diff
@@ -632,7 +632,7 @@ export class PlaygroundCodeEditor extends LitElement {
   private _currentFiletypeSupportsCompletion() {
     // Currently we are only supporting code completion for TS. Change
     // this in a case that we start to support it for other languages too.
-    return this.type === 'ts';
+    return this.type === 'ts' || this.type === 'js';
```

Comment:
Could we also support completions for `tsx` and `jsx`?

Reply:
Yeah I think we could see if it works out of the box with those too